src/key.rs

```diff
@@ -173,18 +172,20 @@ impl From<&RsaPrivateKey> for RsaPublicKey {
 /// Generic trait for operations on a public key.
 pub trait PublicKey: EncryptionPrimitive + PublicKeyParts {
     /// Encrypt the given message.
-    fn encrypt<R: CryptoRngCore>(
+    fn encrypt(
         &self,
-        rng: &mut R,
-        padding: PaddingScheme,
+        rng: &mut impl CryptoRngCore,
+        padding: impl PaddingScheme,
```

Comment:
don't we want to normalize on `impl` vs generics?

Reply:
[dc47326](https://github.com/RustCrypto/RSA/pull/244/commits/dc473264d16b6c59969b3fb1d287c50f4483ac34) should change the remaining usages of `impl Trait` in the PR (back) to generic parameters